docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -2178,6 +2178,60 @@ The third example will autoregister the device and assign the role staff to each
 
 You can have a look in the file vlan_filters.conf, there are some examples on how to use and define filters.
 
+Radius Filter Definition
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+We added the ability to specify filters directly in the portion of code that return the radius answer or do a call to the API.
+
+These rules are only available in one scope:
+
+  returnRadiusAccessAccept
+
+And can be defined using different criteria like:
+
+  node_info.attribute (like node_info.status)
```

Comment:
It would be clearer if written this way IMO : 
node_info.$attribute (like node_info.status)